### DISCUSSION TITLE
OpenAI Realtime API

### DISCUSSION THREAD

**Discussion by CraigOndrak** [semperai/amica#139] [Ideas]
Hi Team,

This is a fantastic project. Any plans to allow connection with OpenAI's new Realtime API or ElevanLabs Conversational AI so all TTS and STT can be bypassed?

**Comment by yume-arasaki** [comment]
No at the moment. We're adding V2V, so it might be easy to add those afterwards. It be cool if somebody writes it and adds it to config though.

**Comment by NavodPeiris** [comment]
this library support OpenAI realtime API with avatars: https://github.com/NavodPeiris/agentic-avatars
